internal/kafka/internal/config/kafka_supported_instance_types.go

```diff
@@ -118,6 +130,17 @@ func (k *KafkaInstanceSize) validate(instanceTypeId string) error {
 		return fmt.Errorf("Kafka instance size '%s' for instance type '%s' specifies a lifespanSeconds seconds value less than or equals to Zero.", k.Id, instanceTypeId)
 	}
 
+	maturityStatusKnown := false
+	for _, status := range getValidMaturityStates() {
+		if k.MaturityStatus == status {
+			maturityStatusKnown = true
```

Comment:
for can be interrupted here:
```suggestion
			maturityStatusKnown = true
			break
```